f2/src/utils/submitReportToServer.js

```diff
@@ -33,8 +39,23 @@ async function submitReportToServer(url = '', data = {}) {
   console.log(`${response.status} (${response.statusText})`)
 }
 
-let data = JSON.parse(JSON.stringify(formDefaults))
-data.contactInfo.fullName = 'Mallory'
-// data.contactInfo.extra = 'extra field!'
+const path = './TestData.json'
 
-submitReportToServer('http://localhost:3000/submit', data)
+fs.access(path, fs.F_OK, (err) => {
+  if (err) {
+    console.error(err)
+    console.info('Using data from JSON Object')
```

Comment:
What's the difference between submit from JSON object and submit from JSON file?

Reply:
This file was originally a little test utility made by steve.  Ignore all the Mallory stuff if you like.

My idea is a) if there is a Test.json file present, use that file and the data inside it to submit to the server.

If that file is not present, load from the defaultvalues.js